lib/cog.ex

```diff
@@ -29,20 +29,20 @@ defmodule Cog do
   def site_namespace, do: "site"
 
   defp build_children(:dev, nochat, _) when nochat != nil do
```

Comment:
Didn't we nuke the `NOCHAT` env var a while back? If so we can drop this whole function head.